### PR TITLE
clean up tiltfile by replacing get_username() func w/ username var

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @jazzdan, @dbentley,

Please review the following commits I made in branch maiamcc/username_var:

63ce7219275b12970ae7be8b1b97790471b88845 (2019-02-21 12:40:44 -0500)
clean up tiltfile by replacing get_username() func w/ username var

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by maiamcc** [comment]
bump!

File: Tiltfile [hunk 1]
```diff
@@ -26,12 +26,11 @@ Here's a quick rundown of these services and their properties:
   * Other notes: Uses yarn. Does a `yarn install` for package dependencies, only if the dependencies have changed
 """
 
-def get_username():
-  return str(local('whoami')).rstrip('\n')
+username() = str(local('whoami')).rstrip('\n')
```
Comment (jazzdan) [inline Tiltfile]: What is this parenthesis syntax in the left hand side?

Comment (maiamcc) [reply Tiltfile]: it's a typo 😅

Comment (maiamcc) [reply Tiltfile]: wow wait why did this work??

Comment (maiamcc) [reply Tiltfile]: ...okay it's heckin' weird that starlark allows that??

Comment (jazzdan) [reply Tiltfile]: Hmm it doesn't in the repl

```
Welcome to Starlark (go.starlark.net)
>>> foo() = "bar"
<stdin>:1:1: can't assign to callexpr
```

Comment (maiamcc) [reply Tiltfile]: wiiiild. wat.

Comment (maiamcc) [reply Tiltfile]: ...
......
..........
i just don't think i ever ran `tilt up` after making this change cuz it def doesn't work DON'T LOOK AT MEEEEE

**Pull request merged**